frontend/src/component/common/Chart/customHighlightPlugin.ts

```diff
@@ -17,10 +17,14 @@ export const customHighlightPlugin = (width = 46, bottomOverflow = 34) => ({
             gradient.addColorStop(0, 'rgba(129, 122, 254, 0)');
             gradient.addColorStop(1, 'rgba(129, 122, 254, 0.12)');
             ctx.fillStyle = gradient;
+
+            const barWidth =
+                (chart.width / (chart.data.labels?.length ?? 1)) *
+                (chart.options.datasets?.bar?.categoryPercentage ?? 1);
```

Comment:
Cool, is categoryPercentage set anywhere or is that a separate PR?

Reply:
No, categoryPercentage is set by default to `0.8`. This is a little wider than the actual bar, which appears to be categoryPercentage * barPercentage (defaults to `0.9`). So this should make it so that the bar is roughly 90% of the highlight width. Ish ...